src/train.hpp

```diff
@@ -0,0 +1,18 @@
+#include <string>
+#include <vector>
+#include "stud.hpp"
+class Trainer
+ {
+ private:
+ std::vector<Student>trainer;
```

Comment:
why does trainer keep students? Normally trainer is not the owner of students. Please rename Trainer class to something like StudentsDatabase.